src/components/MultiHopTrade/components/MultiHopTradeConfirm/MultiHopTradeConfirm.tsx

```diff
@@ -73,7 +73,7 @@ export const MultiHopTradeConfirm = memo(() => {
       previousTradeExecutionState !== confirmedTradeExecutionState &&
       previousTradeExecutionState === TradeExecutionState.FirstHop
     ) {
-      if (isFirstHopOpen) onToggleFirstHop()
+      if (!isFirstHopOpen) onToggleFirstHop()
```

Comment:
Also looks like this was a mistake, though not previously visible since we weren't unmounting and remounting this guy with a new quote